napari/components/viewer_model.py

```diff
@@ -582,8 +584,8 @@ def add_points(
         symbol='o',
         size=10,
         edge_width=1,
-        edge_color='black',
-        face_color='white',
+        edge_color=ColorArray("black"),
```

Comment:
I think I'd prefer the defaults not to be color arrays as I think it could be confusing to people to see function calls inside a function signature. We're able to support users passing strings so I think its fine for them to be the defaults